yt/utilities/exceptions.py

```diff
@@ -742,3 +742,17 @@ def __str__(self):
         msg += "or:\n"
         msg += "  pip install %s\n" % self.module
         return msg
+
+class YTInvalidArgumentType(YTException):
+    def __init__(self, obj_type):
+        self.obj_type = obj_type
+
+    def __str__(self):
+        msg = "One of the arguments to %s (selector) does not match its" \
```

Comment:
Seems odd to mix `+=` and line continuations here.

Reply:
If you want to preserve the newlines, I'd suggest using a triple quoted string. However I tend to not worry about the newlines in messages like this and instead write them like:

```
msg = (
    "This is a long string that I'm breaking up onto multiple lines. "
    "It's valid Python to break strings up like this. Just make sure to "
    "leave spaces when you insert a line break into the code. This works "
    "because two strings next to each other in a Python statement are "
    "implicitly concatenated"
)
```

Reply:
thanks for the nice trick, I will use it!